fdbserver/workloads/ConsistencyCheck.actor.cpp

```diff
@@ -1040,6 +1040,23 @@ struct ConsistencyCheckWorkload : TestWorkload {
 		return true;
 	}
 
+	// Run an empty commit through the system.
+	ACTOR static Future<Void> doEmptyCommit(Database cx) {
+		state Transaction tr(cx);
+		loop {
+			try {
+				wait(::success(tr.getReadVersion()));
```

Comment:
There's a similar function private to ClusterController, without the read version.